test/test_cell.py

```diff
@@ -2,11 +2,10 @@
 # FEniCS Project
 # SPDX-License-Identifier: MIT
 
+import basix
```

Comment:
This (and same in all tests) seems odd. `basix` is the local library name, and `isort` would surely put it back at the bottom of the imports.